src/components/RenderRouter/HeroItem.tsx

```diff
@@ -401,10 +408,19 @@ class HeroItem extends React.Component<Props, State> {
               </div>
             ) : null}
             {this.renderButton(this.state.content.button1, 'heroItemButton')}
-
-            <Link className="inverted" to={this.state.content.link1Action}>
-              {this.state.content.link1Text}
-            </Link>
+            {this.state.content.link1AriaLabel ? (
```

Comment:
I might be wrong, but I don't think this ternary statement necessary.

`link1AriaLabel` will either be a `string` or `undefined`. If it is `undefined`, it will be as if there is no `aria-label`. `""` might an issue, but people shouldn't be inputting an empty string.

Same feedback applies further down.

Reply:
done. Thank you